src/connection/utils.ts

```diff
@@ -0,0 +1,13 @@
+import { ConnectionModuleState } from './ConnectionModule';
+import { Neo4jConnectionModule } from './neo4j/Neo4jConnectionModule';
+import { HiveConnectionModule } from './hive/HiveConnectionModule';
+
+// TODO: get this from the Neodash configuration
```

Comment:
This `TODO` should be resolved before merging.

Reply:
will do